### ISSUE TITLE
expand contractions is not working as expected

### ISSUE THREAD

**Issue by jonanem** [dipanjanS/practical-machine-learning-with-python#17]
When there are more than one single quote(') such as "you'll've" , Expand contractions in text_normalizer.py is  giving output as "you willve" but the expected output is "you will have"

**Comment by dipanjanS** [comment]
Yes sadly this is a known issue even with the contractions package on PyPI. The whole regex pattern framework used for this will need to be modified I am guessing. Will be taking a look at this some time in the future as I get some time.
